packages/tpetra/core/src/Tpetra_Details_DefaultTypes.hpp

```diff
@@ -132,6 +135,14 @@ namespace DefaultTypes {
   };
 #endif
 
+#ifdef KOKKOS_ENABLE_SYCL
+  template<>
+  struct CommBufferMemorySpace<Kokkos::Experimental::SYCLSharedUSMSpace>
```

Comment:
This should actually be specialized on SYCL the execution space. The HIPHostPinnedSpace above is wrong, it should be HIP.

Reply:
Fixed in [`68f57e3` (#9086)](https://github.com/trilinos/Trilinos/pull/9086/commits/68f57e35ee3f00bae7a4de6fd9639c84754e8bc0).